api/eid_oidc_provider/views.py

```diff
@@ -0,0 +1,52 @@
+from django.contrib.auth import REDIRECT_FIELD_NAME
+from django.contrib.sites.shortcuts import get_current_site
+from django.http import HttpResponseRedirect, HttpResponseBadRequest
+from django.template.response import TemplateResponse
+from django.utils.http import is_safe_url
+from django.contrib.auth import login as auth_login
+from django.contrib.auth.models import User
+from django.shortcuts import resolve_url
+from django.conf import settings
+from django.views.decorators.csrf import csrf_exempt
+from .utils.user import username_exists_in_db
+from eid_service.openid_interface import *
```

Comment:
don't use `import *`, [as it hides where symbols come from](https://stackoverflow.com/a/2386740). Also, [explicit is better than implicit](https://www.python.org/dev/peps/pep-0020/).